src/containers/Shared.tsx

```diff
@@ -166,6 +167,7 @@ const Shared = ({ children, ...rest }: Props) => {
     }, [auctionsData, setInternalBalance, setProtInternalBalance])
 
     useEffect(() => {
+        connectWalletActions.setPoolFetchedData(poolData)
         connectWalletActions.setTokensData(tokensData)
     }, [connectWalletActions, tokensData])
```

Comment:
Maybe poolData needs to be added here `[connectWalletActions, tokensData, poolData]`

Reply:
I actually didn't end up using poolData at all--it wasn't being used. I tried adding poolData as an async function being fetched in Shared as well (latest push) but still not working